Sources/Plasma/FeatureLib/pfPython/cyPythonInterface.cpp

```diff
@@ -685,6 +687,11 @@ PYTHON_CLASS_CONVERT_FROM_IMPL(ptOutputRedirector, pyOutputRedirector)
 class pyErrorRedirector
 {
 private:
+    static constexpr char16_t kTracebackTruncatedMarker[5] = {u'[', u'.', u'.', u'.', u']'};
```

Comment:
It might be more readable to do something like

```suggestion
    static constexpr std::u16string_view kTracebackTruncatedMarker{ U"[...]" };
```